tests/commands/test_changelog_command.py

```diff
@@ -453,3 +454,34 @@ def test_changelog_config_start_rev_option(mocker, capsys, config_path):
 
     out, _ = capsys.readouterr()
     assert out == "## Unreleased\n\n### Feat\n\n- after 0.2\n- after 0.2.0\n\n"
+
+
+@pytest.mark.usefixtures("tmp_commitizen_project")
+def test_changelog_incremental_keep_a_changelog_sample_with_annotated_tag(
+    mocker, capsys, changelog_path
+):
+    """Fix #378"""
+    with open(changelog_path, "w") as f:
+        f.write(KEEP_A_CHANGELOG)
+    create_file_and_commit("irrelevant commit")
+    git.tag("1.0.0", annotated=True)
+
+    create_file_and_commit("feat: add new output")
+    create_file_and_commit("fix: output glitch")
+    create_file_and_commit("fix: mama gotta work")
+    create_file_and_commit("feat: add more stuff")
+    create_file_and_commit("Merge into master")
+
+    testargs = ["cz", "changelog", "--incremental"]
+
+    mocker.patch.object(sys, "argv", testargs)
+    # print(git.get_tags())
+    cli.main()
+
+    with open(changelog_path, "r") as f:
+        out = f.read()
+
+    assert (
+        out
+        == """# Changelog\nAll notable changes to this project will be documented in this file.\n\nThe format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),\nand this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).\n\n## Unreleased\n\n### Feat\n\n- add more stuff\n- add new output\n\n### Fix\n\n- mama gotta work\n- output glitch\n\n## [1.0.0] - 2017-06-20\n### Added\n- New visual identity by [@tylerfortune8](https://github.com/tylerfortune8).\n- Version navigation.\n\n### Changed\n- Start using "changelog" over "change log" since it\'s the common usage.\n\n### Removed\n- Section about "changelog" vs "CHANGELOG".\n\n## [0.3.0] - 2015-12-03\n### Added\n- RU translation from [@aishek](https://github.com/aishek).\n"""
+    )
```

Comment:
Wanna try "pytest-regressions"?  You can do so by adding `file_regresion` to this test case and then change the assert function to the following.

```python
file_regression.check(out, extension=".md")
```

The first time you run these tests it will fail as no file exists to be compared. pytest-regressions will generate the file so that you'll succeed the next time you run it. One thing to note is to check whether the output file is what we expect. It will be saved in a folder named "test_changelog_command"